### DISCUSSION TITLE
show_active vs show_active()

### DISCUSSION THREAD

**Discussion by JohnEMagee** [smartcontractkit/full-blockchain-solidity-course-py#1230] [Q&A]
So I ran the initial brownie deploy using network.show_active, but noticed the tutorial then used show_active().  There was no error or discernable difference.  Usually () present or absent make a difference even when empty.  Is this not the case in python or brownie or just a weird quirk?

**Comment by cromewar** [comment]
Hello @JohnEMagee 
Yes there is a difference, as you can see here:
![image](https://user-images.githubusercontent.com/20199180/157474751-999f8774-4e01-4b36-95ed-718bd7a87cc3.png)

Using without the () refer to where the function is, in order to be executed on a context.
Using with () actually calls the function at run time.
